Makefile

```diff
@@ -10,12 +10,13 @@ all: check run-docker
 
 .PHONY: check
 check: venv
-	"$(VENV)"/bin/flake8 *.py
-	"$(VENV)"/bin/mypy --strict conftest.py
+	"$(VENV)"/bin/flake8 *.py **/*.py
+	"$(VENV)"/bin/mypy --strict conftest.py utils
 	"$(VENV)"/bin/reuse lint
 
 .PHONY: run
-run: $(VENV) usbip-runner
+run: export PYNK_DEBUG=10
```

Comment:
I think it should work like this:

> Target-specific variable assignments can be prefixed with any or all of the special keywords export, unexport, override, or private; these apply their normal behavior to this instance of the variable only. 

https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

Otherwise I would probably get a warning due to a missing rule for `export`.

Reply:
Ah, neat! I thought this was just typo